directory.md

```diff
@@ -111,55 +80,51 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
   </li>
 
   <li>
-    <a href="http://bukowskis.com">Bukowskis</a>
-  </li>
-
-  <li>
-    <a href="https://fishbrain.com">Fishbrain</a>
+    <a href="https://bukowskis.com">Bukowskis</a>
   </li>
 
   <li>
-    <a href="http://www.hemnet.se">Hemnet</a>
+    <a href="https://www.doconomy.com">Doconomy</a>
   </li>
 
   <li>
-    <a href="http://www.ibissoft.se">IbisSoft</a>
+    <a href="https://fishbrain.com">Fishbrain</a>
   </li>
 
   <li>
-    <a href="http://www.itsinthenode.com">Its in the Node</a>
+    <a href="https://www.hemnet.se">Hemnet</a>
   </li>
 
   <li>
-    <a href="http://justarrived.se">Just Arrived</a>
+    <a href="https://www.ibissoft.se">IbisSoft</a>
   </li>
 
   <li>
-    <a href="http://www.jmkplay.se">JMK, Stockholms universitet</a>
+    <a href="https://www.itsinthenode.com">Its in the Node</a>
   </li>
 
   <li>
-    <a href="http://www.kollegorna.se">Kollegorna</a>
+    <a href="https://justarrived.se">Just Arrived</a>
   </li>
 
   <li>
-    <a href="http://www.magplus.com">Mag+</a>
+    <a href="https://www.jmkplay.se">JMK, Stockholms universitet</a>
   </li>
 
   <li>
-    <a href="http://www.mynewsdesk.com">Mynewsdesk</a>
+    <a href="https://www.magplus.com">Mag+</a>
```

Comment:
Are they still around?

Reply:
The website is still around, but it looks like it has been bought by MPS Limited in 2016. MPS does not have any Swedish location: https://www.mpslimited.com/locations/. Should be safe to remove as well.